Assets/MRTK/Core/Definitions/MixedRealityExperienceSettingsProfile.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.Tests.EditModeTests")]
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.Tests.PlayModeTests")]
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Experience settings profile for the Mixed Reality Toolkit.
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Mixed Reality Toolkit Experience Settings Profile", fileName = "MixedRealityToolkitExperienceSettingsProfile", order = (int)CreateProfileMenuItemIndices.Configuration)]
+    [HelpURL("https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/configuration/mixed-reality-configuration-guide")]
+    public class MixedRealityExperienceSettingsProfile : BaseMixedRealityProfile
+    {
+        [SerializeField]
+        [Tooltip("The scale of the Mixed Reality experience.")]
+        private ExperienceScale targetExperienceScale = ExperienceScale.Room;
+
+        /// <summary>
+        /// The desired the scale of the experience.
+        /// </summary>
+        public ExperienceScale TargetExperienceScale
+        {
+            get { return targetExperienceScale; }
+            set { targetExperienceScale = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("The height above the floor for the Mixed Reality Experience. 1 unit = 1 meter")]
+        private float floorHeight = 0.0f;
```

Comment:
Is this raising the concept of where the floor is? Or is this the distance from the origin to the floor (ex: -2.0 for a 2m tall person)?

Reply:
The former. At the moment the SceneContentSystem doesn't move the origin, since other platforms (hololens, oculus etc.) move the head position independently from MRTK, making it hard for us manage the playspace/origin/camera relationship in a predictable way.

Reply:
I'm a bit confused here then. If the floor is at 0,0,0 for VR, then it should always be at 0,0,0 correct? If in HoloLens, where is this service calculating the floor distance?

From the test code included, this feels like a content offset vs a floor height.

```
            float originalProfileFloorHeight;

            float floorHeight = 1.3f;
```

This feels a bit backwards (the floor should be a negative distance from the origin when on the head). And reads almost like a content offset. Can we recast conceptually?